src/LocalFooter/LocalFooter.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localFooterColors, localFooterLinkColors } from './LocalFooter.levers';
+import { Container } from '../Container/Container';
+import { Grid } from '../Grid/Grid';
+import { FlexBox } from "../FlexBox/FlexBox";
+import { FlexCell } from "../FlexCell/FlexCell";
+import { Lockup } from "../Lockup/Lockup";
+import Icon from 'react-hero-icon';
+import clsxd from 'clsx-dedupe';
+
+/**
+ * Stanford Local Footer Component.
+ * This usually sits above the Global Footer.
+ *
+ */
+export const LocalFooter = ({ classes = {}, children, ...props }) => {
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // props.color
+  if (props.color && localFooterColors.includes(props.color)) {
+    switch (props.color) {
+      case 'light grey':
+        levers.wrapper = 'su-bg-foggy-light su-text-black';
+        levers.lockup = 'default';
+        break;
+
+      case 'white':
+        levers.wrapper = 'su-bg-white su-text-black';
+        levers.lockup = 'default';
+        break;
+
+      case 'black':
+        levers.wrapper = 'su-bg-black su-text-white';
+        levers.lockup = 'white';
+        break;
+    }
+  }
+
+  // props.link
+  if (props.link && localFooterLinkColors.includes(props.link)) {
+    switch (props.link) {
+      case 'blue':
+        levers.link = '';
+        break;
+
+      case 'red':
+        levers.link = 'su-link-digital-red hover:su-link-black focus:su-link-black';
+        break;
+
+      case 'white':
+        levers.link = 'su-link-white hover:su-link-digital-red-light focus:su-link-digital-red-light';
+        break;
+    }
+  }
+
+  return (
+    <Container width='site' className={clsxd('su-local-footer su-rs-pt-4 su-rs-pb-5', levers.wrapper, classes.wrapper)}>
+      <FlexBox direction='col' gap={true} className={clsxd('md:su-flex-row su-rs-mb-2', classes.header)} justifyContent={'space-between'}>
+        <FlexCell grow={true}>
+          <Lockup color={levers.lockup} line1={props.line1} url={props.url} isLink={props.isLogoLink} />
+        </FlexCell>
+        {props.hasButton && (
+          <FlexCell grow={false}>
+            <a href={props.buttonUrl} className={clsxd('su-inline-block su-bg-digital-red su-text-white su-text-18 hocus:su-text-white hocus:su-bg-archway-dark su-py-9 su-px-20 su-no-underline su-font-regular hover:su-underline focus:su-underline su-rounded-7 su-shadow-md', classes.button)}>
```

Comment:
Missing an option for href. 

Ideally, this would be refactored to use a button/link component and would be passed in as a prop or child.

Reply:
Yeah, was thinking that we'll be using a subcomponent after the CTA link component is done.

Reply:
> Feedback complete.

Thank you for going through this huge PR and the helpful feedback - I'm working on the revision 😃